packages/dx-react-grid/docs/guides/typescript.md

```diff
@@ -0,0 +1,23 @@
+# React Grid - TypeScript
+
+[TypeScript](https://www.typescriptlang.org) is a strict syntactical superset of JavaScript developed by Microsoft. It adds optional static typing to JavaScript and enables catching errors and bugs at build time.
+
+TypeScript became popular and is used in many projects. So we provide React Grid with TypeScript definitions to enable easy and comfortable usage in TypeScript projects.
+
+.embedded-demo(grid-typescript/basic)
+
+## Recommended configuration
```

Comment:
Remove this section. We should avoid embedding information that doesn't relate to the project.